tst/EnergyPlus/unit/EconomicTariff.unit.cc

```diff
@@ -0,0 +1,236 @@
+// EnergyPlus, Copyright (c) 1996-2016, The Board of Trustees of the University of Illinois and
```

Comment:
The new unit test needs to be added to https://github.com/NREL/EnergyPlus/blob/develop/tst/EnergyPlus/unit/CMakeLists.txt